src/model/fxQuotes.js

```diff
@@ -81,9 +157,61 @@ class FxQuotesModel {
 
       await this.validateFxQuoteRequest(fspiopDestination, fxQuoteRequest)
 
+      const envConfig = new Config()
+      if (!envConfig.simpleRoutingMode) {
+        // check if this is a resend or an erroneous duplicate
+        const dupe = await this.checkDuplicateFxQuoteRequest(fxQuoteRequest)
```

Comment:
just wondering about naming: what is `dupe`?

Reply:
shorthand for an object that details wether a request id is a duplicate request and/or if a hash of the request has been seen before.